src/index.html

```diff
@@ -1,40 +1,36 @@
 <!DOCTYPE html>
 <html>
-<head>
-	<title></title>
-	<meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate" />
-	<meta http-equiv="Pragma" content="no-cache" />
-	<meta http-equiv="Expires" content="0" />
-	<script src="bundle.js" type="text/javascript"></script>
-</head>
-
-<body>
-	<div id="container">
-
-	<!-- Start your code from here -->
-	
-
-	<!-- End your code -->
-
-	</div>
-
-	<script type="text/javascript">
-
-	</script>
-
-</body>
-<script>
-/* DO NOT EDIT THIS CODE- WRITTERN TO PREVENT CSS CACHING */	  
-	// Get HTML head element
-	var head = document.getElementsByTagName('HEAD')[0]; 
-	// Create new link Element
-	var link = document.createElement('link');
-	// set the attributes for link element 
-	link.rel = 'stylesheet'; 
-	link.type = 'text/css';
-	link.href = './screen.css?v='+ new Date().getTime(); 
-	// Append link element to HTML head
-	head.appendChild(link); 
-	/* DO NOT EDIT THIS CODE- WRITTERN TO PREVENT CSS CACHING */	  
-</script> 
+    <head>
+        <meta charset="utf-8" />
+        <title>MY FIRST HTML ASSINGMENT BY SWETA BHARTI></title>
+        <link rel= "stylesheet" href= "./index.css" />
+    </head>
+    <body>
+        <section id= "landing-section">
+            <div id = "left-div">
+              
+              <ul>
+                <li class = "list-item">List Item 1</li>
+                 <li class = "list-item">List Item 2</li>
```

Comment:
u have to code only on 14 to 17 page no. , you no need to change anything above or below it

Reply:
its line no. not page no